test/e2e/actions/kube/statuses.go

```diff
@@ -1,43 +1,41 @@
 package kube
 
 import (
+	"k8s.io/apimachinery/pkg/types"
+
 	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1/status"
 	kubecli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kubecli"
 	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
 )
 
 func GetReadyProjectStatus(data *model.TestDataProvider) func() string {
 	return func() string {
-		return kubecli.GetStatusCondition("Ready", data.Resources.Namespace, data.Resources.GetAtlasProjectFullKubeName())
-	}
-}
-
-func GetProjectPEndpointStatus(data *model.TestDataProvider) func() string {
-	return func() string {
-		return kubecli.GetStatusCondition("PrivateEndpointReady", data.Resources.Namespace, data.Resources.GetAtlasProjectFullKubeName())
+		condition, _ := kubecli.GetProjectStatusCondition(data.Context, data.K8SClient, status.ReadyType, data.Resources.Namespace, data.Resources.Project.ObjectMeta.GetName())
+		return condition
 	}
 }
 
-func GetProjectNetworkPeerStatus(data *model.TestDataProvider) func() string {
+func DeploymentReady(data *model.TestDataProvider) func() string {
```

Comment:
Better to swap the name for something like `DeploymentReadyCondition` or even `DeploymentReadyConditionGetter`. With current name one might think this returns a `boolean`.

Reply:
Thanks. Done